pkg/config/config.go

```diff
@@ -97,6 +97,7 @@ func (idConfig *IdentityConfig) loadFromENV() error {
 	loadEnv("TOKEN_DIR", &idConfig.TokenDir)
 	loadEnv("METRICS_SERVER_ADDR", &idConfig.MetricsServerAddr)
 	loadEnv("DELETE_INSTANCE_ID", &idConfig.rawDeleteInstanceID)
+	loadEnv("ENVOY_API_ENABLE", &idConfig.rawEnvoyAPIEnable)
```

Comment:
```suggestion
	loadEnv("TOKEN_SERVER_ENVOY_API", &idConfig.TokenServerEnvoyAPI)
```
follow `TOKEN_SERVER_REST_API` naming conversion?

Reply:
thought the same. though I personally dont like both name (because it does not imply if the data is bool) but to match to the original code `TOKEN_SERVER_REST_API` sounds like a good pick.

Reply:
fixed: https://github.com/AthenZ/k8s-athenz-sia/pull/78/commits/ee4148ff9371e579e5096a60ddb6908811228b67